DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -377,7 +377,22 @@ protocol NewWindowPolicyDecisionMaker {
     }
 
     var webViewDidFinishNavigationPublisher: some Publisher<Void, Never> {
```

Comment:
This is only used for that use case so it should be less risky then filter on the state publisher

In any case if you want we can do this later on it’s in the last commit that I can revert… the other changes still greatly improve the situation.